src/store/DrawToolsStore.js

```diff
@@ -80,6 +86,36 @@ export default class DrawToolsStore {
         }
     };
 
+    onMouseMove = (e) => {
+        e.preventDefault();
+        const vectorType = this.stx.currentVectorParameters.vectorType;
+        const drawingMousePointer = document.getElementById('drawing-mouse-pointer');
+        const container = document.getElementsByClassName('chartContainer primary')[0];
+        const crosshairX = document.getElementsByClassName('stx_crosshair_x')[0];
+        const crosshairY = document.getElementsByClassName('stx_crosshair_y')[0];
+
+        if (vectorType) {
+            drawingMousePointer.style.display = 'block';
+            drawingMousePointer.style.left = `${crosshairX.offsetLeft}px`;
+            drawingMousePointer.style.top = `${crosshairY.offsetTop - 16}px`;
+            container.style.cursor = 'none';
```

Comment:
Interesting.

I think it's better you replace the cursor via CSS instead. In `chart.scss`, `.stx-drag-chart` replaces the cursor with a `*.cur` file.